chart/open-feature-operator/templates/rendered.yaml

```diff
@@ -806,7 +807,7 @@ rules:
 - apiGroups:
   - core.openfeature.dev
   resources:
-  - featureflagconfigurations
+  - FlagSourceConfigurations
```

Comment:
I see deltas in this file after running `make helm-package`. I know that doesn't matter in terms of the build, but maybe we should try to keep it up to date.

Reply:
pushed the deltas